xbps-src

```diff
@@ -198,6 +198,11 @@ $(print_cross_targets)
 
     Supported options can be shown with the 'show-options' target.
 
+-p  <variable,variable2,...>
+    For show target, show specified variables in addition to default ones.
+    Variable is split and each word is print in own line by default. In order to
+    print whole value in one line, append asterisk to variable name.
```

Comment:
either "the whole value" or "all the values"

Reply:
done